src/Tizen.NUI/src/public/WindowSystem/QuickPanelService.cs

```diff
@@ -0,0 +1,167 @@
+﻿/*
+ * Copyright(c) 2017 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Tizen.NUI.WindowSystem.Shell
+{
+    /// <summary>
+    /// Class for the Tizen quickpanel service.
+    /// </summary>
+    /// This class is need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class QuickPanelService : IDisposable
+    {
+        private TizenShell _tzsh;
+        private IntPtr _tzshQpService;
+        private int _tzshWin;
+        private bool disposed = false;
+
+        /// <summary>
+        /// QuickPanel Type.
+        /// </summary>
+        public enum QuickPanelType
+        {
+            /// <summary>
+            /// Unknown
+            /// </summary>
+            Unknown = 0x0,
+            /// <summary>
+            /// System default
+            /// </summary>
+            SystemDefault = 0x1,
+            /// <summary>
+            /// Context menu
+            /// </summary>
+            ContextMenu = 0x2,
+        }
+
+        /// <summary>
+        /// Creates a new Quickpanel Service handle.
+        /// </summary>
+        /// <param name="tzShell">The TzShell instance.</param>
+        /// <param name="win">The window to provide service of the quickpanel.</param>
+        /// <param name="type">The type of quickpanel service.</param>
+        /// <returns>A new Quickpanel Service instance.</returns>
```

Comment:
Don't use `<returns>` tag with constructor method.

Reply:
I remove <returns> in constructor.